css/style.css

```diff
@@ -0,0 +1,101 @@
+/*
+Project Name: TIC-TAC-TOE
+Author: Jonathan Andrew
+Developer @GA in Seattle
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+
+#test {
+  height: 500px;
+}
+/* Box Model Hack */
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:after {
+     content: ".";
+     display: block;
+     clear: both;
+     visibility: hidden;
+     line-height: 0;
+     height: 0;
+}
+
+.clear {
+  clear: both;
+}
+
+.alignright {
+  float: right;
+  padding: 0 0 10px 10px; /* note the padding around a right floated image */
+}
+
+.alignleft {
+  float: left;
+  padding: 0 10px 10px 0; /* note the padding around a left floated image */
+}
+
+.hide {
+  display: none;
+}
+
+
+
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+body{
+	color: #ffffff;
+	background-color: #000000;
+}
+
+#container{
+	width: 610px;
```

Comment:
Im not sure about your browser, but in mine it was having issues forcing the squares into this container. You could take out the width and make it display: inline-block in order to make it a flexible container. The only thing is if you want to center it you'd need to wrap it with another div, and then make its margin 0 auto to center it within that other container.